src/modules/common/containers/image-selector.jsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { Thumbnail, FileButton } from 'zooniverse-react-components';
+import toBlob from 'data-uri-to-blob';
```

Comment:
Not sure why it didn't get in there, but it's working because it got installed with another dependency. Updated so that it's explicitly in the dep list.